apps/lib/publisher_agent/config.go

```diff
@@ -86,11 +84,18 @@ func LoadConfig(configFilePath string, keysFilePath string) (*StorkPublisherAgen
 	}
 
 	var keysFile KeysFile
```

Comment:
Can we change the name of this struct? I feel like instead of KeysFile it should be `Secrets` and the other struct should be `Config`

Reply:
I named it keys initially because some of the data isn't actually secret (e.g. public keys), but `secrets` is fine too. we should probably move the public keys into config eventually but let's hold off on that for now since users will need to update their config files

Reply:
I agree, although there is still public info in there (pub keys, I assume oracle id is public as well). Should the public stuff be moved into config? Or are we okay with secrets not being only secrets?

Reply:
https://github.com/Stork-Oracle/stork-external/pull/34#discussion_r1811269653

nvm, @harryrackmil actually already answered this. I'm fine to just do the renaming but keep the structure for now.

Reply:
I'm more commenting on the notion of it being a "file" than anything, so happy to continue calling it keys for now

Reply:
For now settled on calling it "Keys" for slightly higher accuracy and consistency